docs/performance/inserts/testing.txt

```diff
@@ -0,0 +1,96 @@
+.. _testing_inserts_performance:
+
+===========================
+Testing Inserts Performance
+===========================
+
+Every setup is different. The best way to tune your particular setup is to do
+performance testing.
+
+The easiest way to do this is to use the `cr8`_ tool.
+
+First of all, create a table specifically for performance testing:
+
+.. code-block:: psql
+
+    cr> CREATE TABLE my_table (
+    ...   id integer,
+    ...   name string
+    ... );
+    CREATE OK, 1 row affected  (... sec)
+
+This is a simple example. You should create a table that mirrors the sort of
+data you plan to use in production.
+
+Next, generate some fake data:
```

Comment:
Ideally they'd use actual data that they plan to store - this should be as a fallback used with caution because the data will very likely be different

Reply:
done. added whole new section